### PR TITLE
Clarify PR Template settings behavior by git provider

### LINKED ISSUE #2742: Clarify the PR Template URL settings by connection type
### Contributions

- [X] I have read the contribution docs, and understand what's expected of me.

### Link to the page on docs.getdbt.com requiring updates

https://docs.getdbt.com/docs/collaborate/git/pr-template

### What part(s) of the page would you like to see updated?

The top bit!

There was a user who was not sure why they were not seeing a PR option, and it turned out it was due to their connection method. 

In chatting with the IDE team, we clarified the current logic, and have a few bullet points to make the behavior a little clearer!

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by dave-connors-3**
## What are you changing in this pull request and why?

Description of how and when to expect the `Create pull request` prompt in the IDE

Closes #2742

### CODE REVIEW

File: website/docs/docs/collaborate/git/pr-template.md [hunk 1]
```diff
@@ -2,17 +2,22 @@
...
-instead show a prompt to merge the changes directly into the `master` branch.
+Account Settings. If this setting is blank, the IDE will prompt users to merge the changes directly into their default branch.
 
 <Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/configure-template-url-new-ide.png" title="Open a PR in the IDE"/>
 
+### PR Template URL by git provider
+
+The PR Template URL setting will be automatically set for most repositories, depending on the connection method.
+
+- If you are connecting to your repository using in-app integrations with your git provider or using the `Git Clone` method via SSH, this URL setting will be automatically populated and can be edited.
```
Comment (mirnawong1) [inline website/docs/docs/collaborate/git/pr-template.md]: suggesting active voice 

```suggestion
- If you connect to your repository via in-app integrations with your git provider or the "Git Clone" method via SSH, this URL setting will be auto-populated and editable.
```

File: website/docs/docs/collaborate/git/pr-template.md [hunk 2]
```diff
@@ -2,17 +2,22 @@
...
+ - If you are using a dbt Cloud Managed repository, this URL will not be set, and IDE will prompt users to merge the changes directly into their default branch.
```
Comment (mirnawong1) [inline website/docs/docs/collaborate/git/pr-template.md]:
```suggestion
 - If you connect via a dbt Cloud [Managed repository](/docs/collaborate/git/managed-repository), this URL will not be set, and the IDE will prompt users to merge the changes directly into their default branch.
```

**Review by mirnawong1 — COMMENTED** [review]
hey @dave-connors-3 thanks for opening this up! I've left a few tweaks and let me know what you think!

**Comment by dave-connors-3** [comment]
@mirnawong1 took both excellent suggestions! been battling the passive voice since 9th grade

**Review by mirnawong1 — APPROVED** [review]
good to go @dave-connors-3 thank you so much!

team passive voice here too 😅

**Pull request merged**